src/Integration.Vsix.UnitTests/ErrorList/IssuesSnapshotTests _GetValue.cs

```diff
@@ -94,31 +94,44 @@ public void TryCreateDetailsStringContent_ReturnsIssueMessage()
         [TestMethod]
         public void GetValue_UnknownColumn_Null()
         {
-            AssertGetValueReturnsNull(columnName: "asdsdgdsgrgddfgfg");
+            AssertGetValueReturnsFalse(columnName: "asdsdgdsgrgddfgfg");
         }
 
         [TestMethod]
         [DataRow(-1)]
         [DataRow(999)]
         public void GetValue_IndexOutOfRange_Null(int index)
         {
-            AssertGetValueReturnsNull(index);
+            AssertGetValueReturnsFalse(index);
         }
 
         [TestMethod]
-        public void GetValue_IssueHasNoSpan_Null()
+        public void GetValue_IssueHasNoSpanKeyIsLine_Null()
```

Comment:
The test name isn't easy to read. An extra `_` would help i.e. `GetValue_IssueHasNoSpan_KeyIsLine_Null`.
Same for the next test.

Reply:
... and the two tests could be combined into a single test with two DataRows, couldn't it?